src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -23,15 +23,77 @@
 	<NcAppNavigation :aria-label="t('spreed', 'Conversation list')">
 		<div class="new-conversation"
 			:class="{ 'new-conversation--scrolled-down': !isScrolledToTop }">
-			<SearchBox v-model="searchText"
+			<SearchBox ref="searchbox"
+				v-model="searchText"
 				class="conversations-search"
+				:class="{'conversations-search--expanded': isFocused,
+					'conversations-search--filter' : isFiltered }"
 				:is-searching="isSearching"
+				@blur="setIsFocused(false)"
+				@focus="setIsFocused(true)"
 				@input="debounceFetchSearchResults"
 				@submit="onInputEnter"
 				@keydown.enter.native="handleEnter"
 				@abort-search="abortSearch" />
-			<NewGroupConversation v-if="canStartConversations" />
+
+			<!-- Options -->
+			<div class="options"
+				:class="{'options--hidden': isFocused}">
+				<NcActions ref="filterMainBtn"
+					class="filters-button">
+					<template #icon>
+						<FilterIcon v-if="!isFiltered" :size="15" />
+						<FilterCheckIcon v-else-if="isFiltered" :size="15" />
```

Comment:
```suggestion
						<FilterCheckIcon v-else :size="15" />
```
That's also works, though I'd change them, to avoid '!' negation